src/pages/workspace/accounting/WorkspaceAccountingPage.tsx

```diff
@@ -13,19 +14,27 @@ import Section from '@components/Section';
 import ThreeDotsMenu from '@components/ThreeDotsMenu';
 import type ThreeDotsMenuProps from '@components/ThreeDotsMenu/types';
 import useLocalize from '@hooks/useLocalize';
+import usePermissions from '@hooks/usePermissions';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 // import useWaitForNavigation from '@hooks/useWaitForNavigation';
 import useWindowDimensions from '@hooks/useWindowDimensions';
+import {hasAccessToAccountingFeatures} from '@libs/WorkspacesSettingsUtils';
+import Navigation from '@navigation/Navigation';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
 import type {AnchorPosition} from '@styles/index';
 import CONST from '@src/CONST';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
 
-function WorkspaceAccountingPage() {
+function WorkspaceAccountingPage({policy}: WithPolicyProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     // const waitForNavigate = useWaitForNavigation();
     const {isSmallScreenWidth, windowWidth} = useWindowDimensions();
+    const {canUseAccountingIntegrations} = usePermissions();
+    const hasAccess = hasAccessToAccountingFeatures(policy, canUseAccountingIntegrations);
```

Comment:
```suggestion
    const hasAccess = PolicyUtils.isPolicyAdmin(policy) && policy?.areConnectionsEnabled && canUseAccountingIntegrations;
```
We don't need a new function.

Reply:
We need to use same logic on `WorkspaceInitialPage` so we don't push the menu item to `protectedCollectPolicyMenuItems` in the first place

Reply:
Hi @s77rt, `hasAccessToAccountingFeatures` uses same logic as your suggestion, but i can commit that change as well.